### PR TITLE
Create Step Three Form register

### PR DESCRIPTION
**Pull request by AFahgan**
image

### CODE REVIEW

File: client/src/Components/RegisterPageComponents/Stepper/Steps/gymdata.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+const gymData = {
```
Comment (AhmedQeshta) [inline client/src/Components/RegisterPageComponents/Stepper/Steps/gymdata.jsx:1]: you can use a file JSON form `client\src\Services\Cities.json` , and and import it like this 
```js
import * as Cities from "../../../../Services/Cities.json";

const { cities } = Cities;
```
and add other like above

**Review by AhmedQeshta — CHANGES_REQUESTED** [review]

File: client/src/Components/RegisterPageComponents/Stepper/BodyStepper.jsx [hunk 2]
```diff
@@ -26,18 +33,31 @@ export default function BodyStepper() {
       description: "",
     },
     validationSchema: SignupSchema,
+    onSubmit: (values) => {
+      console.log(values);
+      dispatch(handleNext());
+    },
+  });
+  const thirfForm = useFormik({
```
Comment (LinaYahya) [inline client/src/Components/RegisterPageComponents/Stepper/BodyStepper.jsx:41]:
```suggestion
  const detailsForm = useFormik({
```

File: client/src/Components/RegisterPageComponents/Stepper/BodyStepper.jsx [hunk 3]
```diff
@@ -26,18 +33,31 @@ export default function BodyStepper() {
...
   });
-
   let StepComponent;
   switch (activeStep) {
     case 1:
       StepComponent = <ContactInformation contactForm={contactForm} />;
       break;
     case 2:
-      StepComponent = <StepThreeComponent />;
+      StepComponent = <StepThreeComponent thirfForm={thirfForm} />;
```
Comment (LinaYahya) [inline client/src/Components/RegisterPageComponents/Stepper/BodyStepper.jsx:60]:
```suggestion
      StepComponent = <StepThreeComponent detailsForm={detailsForm} />;
```

File: client/src/Components/RegisterPageComponents/Stepper/BodyStepper.jsx [hunk 4]
```diff
@@ -1,22 +1,29 @@
...
 const SignupSchema = Yup.object().shape({
   city: Yup.string().required("حقل المدينة مطلوب"),
   phone: Yup.string()
     .length(10, "رقم الهاتف غير صحيح")
     .required("حقل رقم الهاتف مطلوب"),
   description: Yup.string().max(255, "الوصف يجب أن لا يزيد عن 255 حرف"),
+  features: Yup.array().min(1, "حقل المزايا مطلوب"),
+  gender: Yup.string().required("حقل الفئة مطلوب"),
+  monthPrice: Yup.number().moreThan(1, "قيمة الاشتراك الشهرية مطلوبة"),
+  sixMonthPrice: Yup.number().moreThan(1, "قيمة اشتراك الستة أشهر مطلوب"),
```
Comment (LinaYahya) [inline client/src/Components/RegisterPageComponents/Stepper/BodyStepper.jsx:21]: remove required from textField and add your validation here
```suggestion
  sixMonthPrice: Yup.number().moreThan(1, "قيمة اشتراك الستة أشهر مطلوب").required("msg"),
,
```
the same for month price validation

File: client/src/Components/RegisterPageComponents/Stepper/Steps/StepThreeComponent.jsx [hunk 5]
```diff
@@ -1,21 +1,169 @@
...
+          id="gym-name-required"
+          label="اشتراك ستة أشهر"
+          type="text"
+          name="sixMonthPrice"
+          variant="outlined"
+          onChange={thirfForm.handleChange}
+          value={thirfForm.values.sixMonthPrice}
+          error={!!thirfForm.errors.sixMonthPrice}
+          helperText={thirfForm.errors.sixMonthPrice}
+          required
```
Comment (LinaYahya) [inline client/src/Components/RegisterPageComponents/Stepper/Steps/StepThreeComponent.jsx:108]:
```suggestion
          
```

File: client/src/Components/RegisterPageComponents/Stepper/Steps/StepThreeComponent.jsx [hunk 6]
```diff
@@ -1,21 +1,169 @@
...
+          id="gym-name-required"
+          label="الاشتراك الشهري"
+          type="text"
+          name="monthPrice"
+          variant="outlined"
+          onChange={thirfForm.handleChange}
+          value={thirfForm.values.monthPrice}
+          error={!!thirfForm.errors.monthPrice}
+          helperText={thirfForm.errors.monthPrice}
+          required
```
Comment (LinaYahya) [inline client/src/Components/RegisterPageComponents/Stepper/Steps/StepThreeComponent.jsx:95]:
```suggestion
```

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by rawaneyad — APPROVED** [review]
Great Job Ayman 👍🏼

**Review by AhmedQeshta — APPROVED** [review]

**Pull request merged**